Algorithms/Sorting/Java/BucketSort.java

```diff
@@ -0,0 +1,62 @@
+//Bucket Sort in java
+
+import java.util.*;
+
+
+public class BucketSort {
+
+
+     //Driver method
+	public static void main(String[] args) {
+		
+
+        double a[] = {0.33,0.756,0.146,0.94,0.32,0.545};
```

Comment:
It would have been better that instead of hardcoding the values, they are taken as input.